<?xml version="1.0"?>

<launch>   

<node name="cloudlistener_toMap_node" pkg="slammin" type="cloudlistener_toMap" respawn="false" output="screen"/>
<node name="slammin_matcher_node" pkg="slammin" type="slammin_matcher" respawn="false" output="screen"/>

</launch>
